tools/webtransport/h3/webtransport_h3_server.py

```diff
@@ -0,0 +1,247 @@
+import asyncio
+import logging
+import os
+import signal
+import sys
+import traceback
+from urllib.parse import urlparse
+from typing import Any, Dict, List, Optional, Tuple
+
+from aioquic.asyncio import QuicConnectionProtocol, serve
+from aioquic.h3.connection import H3_ALPN, H3Connection
+from aioquic.h3.events import H3Event, HeadersReceived, WebTransportStreamDataReceived, DatagramReceived
+from aioquic.quic.configuration import QuicConfiguration
+from aioquic.quic.connection import stream_is_unidirectional
+from aioquic.quic.events import QuicEvent, ProtocolNegotiated
+from aioquic.tls import SessionTicket
+from aioquic.quic.packet import QuicErrorCode
+
+SERVER_NAME = 'webtransport-h3-server'
+logger = logging.getLogger(__name__)
+
+handlers_path = ''
+
+
+class WebTransportH3Protocol(QuicConnectionProtocol):
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        self._handler: Optional[EventHandler] = None
+        self._http: Optional[H3Connection] = None
+
+        def handle_signal(*_):
```

Comment:
Why install a signal handler here, rather than doing the handling in the default `KeyboardInterrupt` exception handler in the main loop?

Reply:
Removed.